src/qonnx/core/modelwrapper.py

```diff
@@ -145,6 +146,31 @@ def transform(self, transformation, make_deepcopy=True, cleanup=True):
             (transformed_model, model_was_changed) = transformation.apply(transformed_model)
         if cleanup:
             transformed_model.cleanup()
+
+        if apply_to_subgraphs:
```

Comment:
minor: application of the transform to subgraphs always seems to come after tha transform has been applied to the original graph itself. are there any cases where this ordering will cause trouble?

Reply:
makes sense, let's keep it as is and address it if it becomes a problem.